File: 028/028.js
const a = n =>  Math.floor(n * (n + 2) / 4) + Math.floor((n % 4) / 3) + 1

const n = 1001
let sum = 0
for (let i = 1; a(i) <= n * n; i++)
  sum += a(i)
console.log(sum)

File: 028/028.c
#include <stdio.h>

int a(int n) {
    return n*(n+2)/4 + (n%4)/3 + 1;
}

int main() {
    int n = 1001;
    int sum = 0;
    for (int i = 1; a(i) <= n*n; i++)
        sum += a(i);
    printf("%d\n", sum);
}
